### PR TITLE
TW 437 Desktop header updates

### PR DESCRIPTION
**Pull request by HeatherStenson**
Changes for the settings updates are happening in https://github.com/postmanlabs/postman-docs/pull/4065

### CODE REVIEW

**Comment by HeatherStenson** [comment]
I'm marking this as a draft for now since there might be some changes as the launch gets closer!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ _Workspace as an entity_ represents a whole container where being an admin gives
 
 To create a new workspace, select **Workspaces** in the header, then select **Create Workspace**.
 
-<img alt="Create new workspace" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.1.jpg" width="300px"/>
+<img alt="Create new workspace" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.19.jpg" width="300px"/>
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/creating-workspaces.md:58]: Roger that!

**Review by claireriley — APPROVED** [review]
LGTM! ☀️

File: src/pages/docs/getting-started/syncing.md [hunk 2]
```diff
@@ -29,17 +29,16 @@ You can have up to three devices logged in to your Postman account simultaneousl
 
 ## Understanding sync states
 
-Postman indicates the sync state of your workspace with the sync icon. Possible states are:
+Postman indicates the sync state of your workspace with the sync icon in the Postman footer. Hover over the sync icon to see more information about your connection and sync status.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/syncing.md:32]:
```suggestion
Postman indicates the sync state of your workspace with the sync icon in the footer. Hover over the sync icon to see more information about your connection and sync status.
```
```suggestion
Postman indicates the sync state of your workspace with the sync icon in the Postman footer. Hover over the sync icon to see more information about your connection and sync status.
```

Comment (HeatherStenson) [inline src/pages/docs/getting-started/syncing.md:32]:
```suggestion
Postman indicates the sync state of your workspace with the sync icon in the footer. Hover over the sync icon to see more information about your connection and sync status.
```
```suggestion
Postman indicates the sync state of your workspace with the sync icon in the Postman footer. Hover over the sync icon to see more information about your connection and sync status.
```

**Pull request merged**